Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/Allocator/Jobs/DriverFeeUpdates/DriverFee.hs

```diff
@@ -404,3 +405,25 @@ scheduleJobs transporterConfig startTime endTime merchantId maxShards = do
         startTime = startTime,
         endTime = endTime
       }
+  createJobIn @_ @'SendOverlay dfCalculationJobTs maxShards $
+    SendOverlayJobData
+      { merchantId = merchantId,
+        rescheduleInterval = Nothing,
+        overlayKey = manualInvoiceGeneratedNudgeKey,
+        udf1 = Just $ show MANUAL,
+        condition = InvoiceGenerated MANUAL
+      }
+  createJobIn @_ @'SendOverlay dfCalculationJobTs maxShards $
+    SendOverlayJobData
+      { merchantId = merchantId,
+        rescheduleInterval = Nothing,
```

Comment:
Is this field always Nothing? Can we remove it?

Reply:
No we cannot remove it. This Job was created via code ( not via dashboard ) - and here we don't want to reschedule it. As always it will scheduled when notification scheduler is scheduled.

But for the case of scheduling the Job via dashboard we require this. From there we can get reschedule interval as "Just 86400" means reschedule the entire job after 1 day.